packages/block-library/src/table-of-contents/edit.js

```diff
@@ -212,7 +213,10 @@ export default function TableOfContentsEdit( {
 
 	return (
 		<>
-			<nav { ...blockProps }>
+			<nav
+				{ ...blockProps }
+				aria-label={ ariaLabel || __( 'Table of Contents' ) }
+			>
```

Comment:
```suggestion
			<nav { ...blockProps }>
```

I think this change is unnecessary, as it overrides the block's default aria-label, "Block: Table of Contents".

Reply:
I've removed this change in https://github.com/WordPress/gutenberg/pull/71586/commits/9bd08b773dd472316918d381733a25836f91b17d.